generated/alerting/api_alerting.go

```diff
@@ -583,7 +583,7 @@ func (a *AlertingApiService) CreateRuleExecute(r ApiCreateRuleRequest) (*RuleRes
 				} else {
 					key = apiKey.Key
 				}
-				localVarHeaderParams["ApiKey"] = key
+				localVarHeaderParams["Authorization"] = key
```

Comment:
This generated Kibana client code is buggy :(

We will have to revisit this and ensure we can generate functioning HTTP requests from the OpenAPI specs. I think this task is beyond this PR so I've modified the generated code directly. We should open an issue for fixing this!